Modules/Commvault.CommCell/Commvault.CommCell.psm1

```diff
@@ -1001,6 +1001,251 @@ function Set-CVClient {
     }
 }
 
+function Get-CVClientLicense {
+<#
+.SYNOPSIS
+    Method to get licenses for a client.
+
+.DESCRIPTION
+    Method to get licenses for a client.
+
+.PARAMETER Name
+    Get licenses on client specified by Name.
+
+.PARAMETER Id
+    Get licenses on client specified by Id.
+
+.EXAMPLE
+    Get-CVClientLicences -Name 'carbonwincs1'
+
+.OUTPUTS
+    [PsCustomObject] containing details of the license and the plaform type
+
+.NOTES
+    Author: Craig Tolley
+    Based on API Documentation at https://documentation.commvault.com/commvault/v11/article?p=47685.htm
+#>
+    [CmdletBinding(DefaultParameterSetName = 'Default')]
+    [OutputType([PSCustomObject])]
+    param(
+        [Alias('Client')]
+        [Parameter(Mandatory = $False, ParameterSetName = 'ByName')]
+        [ValidateNotNullorEmpty()]
+        [String] $Name,
+
+        [Alias('ClientId')]
+        [Parameter(Mandatory = $False, ParameterSetName = 'ById')]
+        [ValidateNotNullorEmpty()]
+        [Int32] $Id
+    )
+    
+    begin { Write-Debug -Message "$($MyInvocation.MyCommand): begin"
+
+        try {
+            $sessionObj = Get-CVSessionDetail $MyInvocation.MyCommand.Name
+            $endpointSave = $sessionObj.requestProps.endpoint
+
+            if ($PSCmdlet.ParameterSetName -eq 'ByName' -or
+                $PSCmdlet.ParameterSetName -eq 'ById' ) {
+                $foundClient = $False
+            }
+            else {
+                $foundClient = $null
+            }
+        }
+        catch {
+            throw $_
+        }
+    }
+    
+    process { Write-Debug -Message "$($MyInvocation.MyCommand): process"
+
+        try {
+            $sessionObj.requestProps.endpoint = $endpointSave
+
+            if ($PSCmdlet.ParameterSetName -eq 'ByName') {
+                $Id = (Get-CVClient -Name $Name).clientId
+            }
+
+            $sessionObj.requestProps.endpoint = $sessionObj.requestProps.endpoint -creplace ('{clientId}', $Id)
+            $headerObj = Get-CVRESTHeader $sessionObj
+            $body = ''
+            $payload = @{ }
+            $payload.Add('headerObject', $headerObj)
+            $payload.Add('body', $body)
+            $validate = 'licensesInfo'
+    
+            $response = Submit-CVRESTRequest $payload $validate
+    
+            if ($response.IsValid) {
+                Write-Output $response.Content.licensesInfo
+            }
```

Comment:
if ($response.IsValid) {
                Write-Output $response.Content.licensesInfo
            }
            else {
                Write-Information -InformationAction Continue -MessageData "INFO: $($MyInvocation.MyCommand): license information not found for client Id [$Id]"
            }